### ISSUE TITLE
problems about tf.nn.softmax

### ISSUE THREAD

**Issue by DewenXu** [tensorflow/tensorflow#23426]
I need a softmax function like this:

def softmax(a):
    c = np.max(a)
    exp_a = np.exp(a - c)
    sum_exp_a = np.sum(exp_a)
    y = exp_a / sum_exp_a
    return y 

please help
Thanks!

**Comment by joeyearsley** [comment]
This is better asked on stackoverflow in future.

```
def softmax(a):
  c = tf.reduce_max(a)
  exp_a = tf.exp(a - c)
  sum_exp_a = tf.reduce_sum(exp_a)
  y = exp_a / sum_exp_a
  return y
```

**Comment by harshini-gadige** [comment]
@DewenXu  -  Hi, if this is a feature, please use this template and open a new issue with all the information requested in the template. If it's a question, please post to Stack Overflow.
